### PR TITLE
[DF] Add tutorial of connection to Dask

### PR DESCRIPTION
**Pull request by vepadulano**

### CODE REVIEW

File: tutorials/dataframe/distrdf002_dask_connection.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import ROOT
+
+# Point RDataFrame calls to Dask RDataFrame object
+RDataFrame = ROOT.RDF.Experimental.Distributed.Dask.RDataFrame
+
+# Setup connection to a Dask cluster. Two ingredients are needed:
+# 1. Creating a cluster object that represents computing resources. This can be
+#    done in various ways depending on the type of resources at disposal. To use
+#    only the local machine (e.g. your laptop), a `LocalCluster` object cam be
+#    used. This step is optional if the cluster resources are setup externally.
```
Comment (eguiraud) [inline tutorials/dataframe/distrdf002_dask_connection.py:28]: > This step is optional if the cluster resources are setup externally.

I am worried users will have no idea what that means

File: tutorials/dataframe/distrdf002_dask_connection.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+#    http://distributed.dask.org/en/stable/api.html#cluster .
+# 2. Creating a Dask client object that connects to the cluster. This accepts
+#    directly the object previously created. In case the cluster was setup
+#    externally, you need to provide an endpoint URL to the client, e.g.
+#    'https://myscheduler.domain:8786'.
+
+# In this example, a cluster object is created for the local machine, using
+# multiprocessing (processes=True) on 4 workers (n_workers=4) each using only
+# 1 core (threads_per_worker=1).
+cluster = LocalCluster(n_workers=4, threads_per_worker=1, processes=True)
```
Comment (eguiraud) [inline tutorials/dataframe/distrdf002_dask_connection.py:39]: can we also have examples, in this tutorial, for condor, slurm, ssh? either commented or uncommented but unused -- so we can point users here to show how it's done, and from here they know what dask docs they have to further read if they want to know more

File: tutorials/dataframe/distrdf002_dask_connection.py (hunk 1 above)
Comment (vepadulano) [reply tutorials/dataframe/distrdf002_dask_connection.py:28]: what about
` This step is optional if the cluster resources are managed by an administrator and they provided you with a URL to connect to the Dask scheduler` ?

Comment (etejedor) [reply tutorials/dataframe/distrdf002_dask_connection.py:28]: or `This step can be skipped if you have access to an existing Dask cluster; in that case, the cluster administrator should provide you with a URL to connect to such cluster in Step 2.`

File: tutorials/dataframe/distrdf002_dask_connection.py (hunk 2 above)
Comment (etejedor) [reply tutorials/dataframe/distrdf002_dask_connection.py:39]: Perhaps commented, since the tutorial needs to be runnable?

**Comment by vepadulano** [comment]
Comments addressed, see the updated docs

**Review by etejedor — APPROVED** [review]

**Pull request merged**